asv_bench/benchmarks/io/csv.py

```diff
@@ -44,6 +40,9 @@ def setup_cache(self, test_filename="io_test_file"):
         return test_filenames
 
     def setup(self, test_filenames, shape, *args, **kwargs):
+        # ray init
+        if ASV_USE_IMPL == "modin":
+            pd.DataFrame([])
```

Comment:
We should probably have a `modin.init` instead of using this 😄. In another PR of course.

Reply:
I agree:)

Create the issue for this #2989.